packages/ndla-ui/src/glossary/Glossary.jsx

```diff
@@ -21,75 +19,34 @@ const sourceClasses = new BEMHelper({
   prefix: 'c-',
 });
 
-class Glossary extends Component {
-  constructor(props) {
-    super(props);
-    this.state = { isOpen: false };
-    this.handleClick = this.handleClick.bind(this);
-    this.close = this.close.bind(this);
-  }
-
-  componentDidMount() {}
-
-  handleClick() {
-    this.setState({ isOpen: !this.state.isOpen });
-  }
-
-  close() {
-    this.setState({ isOpen: false });
-  }
-
-  render() {
-    const { children } = this.props;
-    const { isOpen } = this.state;
-
-    return (
-      <div {...classes('item')}>
-        <button
-          aria-haspopup
-          href="#test"
-          {...classes('link')}
-          onClick={this.handleClick}>
-          {children}
-        </button>
-        {isOpen
-          ? <div {...classes('popup')}>
-              <button
-                {...classes('close', '', 'u-close')}
-                onClick={this.handleClick}>
-                Lukk
-              </button>
-              <h3 {...classes('title')}>
-                {children}
-              </h3>
-              <p {...classes('description')}>
-                {this.props.definition}
-              </p>
-              <div {...sourceClasses()}>
-                <ClickableLicenseByline
-                  className="c-source-list__item"
-                  noText
-                  license={[BY, NC, ND]}
-                />
-                <span {...sourceClasses('item')}>
-                  {this.props.author}
-                </span>
-                <span {...sourceClasses('item')}>
-                  {this.props.source}
-                </span>
-              </div>
-            </div>
-          : null}
-      </div>
-    );
-  }
-}
+const Glossary = ({ title, authors, content, children }) =>
+  <span {...classes('item')}>
+    <button aria-haspopup {...classes('link')}>
+      {children}
+    </button>
+    <span {...classes('popup')}>
+      <button {...classes('close', 'u-close')}>Lukk</button>
+      <span {...classes('title')}>
+        {title}
+      </span>
+      <span {...classes('content')}>
+        {content}
+      </span>
+      <span {...sourceClasses()}>
```

Comment:
Mulig vi diskuterte det, men finnes det noe lisensinformasjon på et begrep?

Reply:
[Ikke etter det jeg kan se](http://search.ndla.no/nb/ndla_search?language%5B0%5D=nb&language%5B1%5D=und&site=ndla&f%5Bim_vid_100016%5D%5B%5D=76656&f%5Bim_vid_100004%5D%5B%5D=71963). Jeg kan spørre på neste "spolemøte" for å være helt sikker

Reply:
Siden de vises i Netlife designet så bør vi ha de i koden inntil videre. Kanskje du bare kan sende inn lisensen på selve artikkelen?

Reply:
Ok, da kan jeg bare legge på lisensen til artikkelen, ja.